pkg/dataobj/metastore/iter.go

```diff
@@ -185,13 +186,25 @@ func forEachStreamSectionPointer(
 						}
 						buf[rIdx].UncompressedSize = values.Value(rIdx)
 					}
+				case pointers.ColumnTypeColumnName:
```

Comment:
Unfortunately, I don't think this will get any results
The rows containing stream stats don't have column name populated in the pointers section.
They're only populated for bloom filter rows to identify which column the bloom filter applies to, and bloom filter rows are mutually exclude to stream stats rows in this section.
This is a side effect of the pointers section being a wide section with many types of columns used for different purposes, and definitely isn't clear from the code